scripts/install-latest.sh

```diff
@@ -106,9 +106,20 @@ yarn_install() {
   printf "${white}Installing Yarn!$reset\n"
 
   if [ -d "$HOME/.yarn" ]; then
-    printf "$red> ~/.yarn already exists, possibly from a past Yarn install.$reset\n"
-    printf "$red> Remove it (rm -rf ~/.yarn) and run this script again.$reset\n"
-    exit 0
+    if [ -n `which yarn` ]; then
+      LATEST_VERSION=`curl https://yarnpkg.com/latest-version`
+      YARN_VERSION=`yarn -V`
+
+      if [ "$LATEST_VERSION" -eq "$YARN_VERSION" ]; then
+        printf "$green> Yarn is already at the latest version.$reset\n"
+      else
+        rm -rf "$HOME/.yarn"
```

Comment:
2scary4me, maybe warn the user before doing this.

"A newer version of Yarn is available, do you want to delete the existing version and upgrade? (Y/N)"